keyboards/preonic/keymaps/pitty/keymap.c

```diff
@@ -0,0 +1,236 @@
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, TO(_GAME), TO(_QWERTY)  \
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|AudOff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|MusOff|MidiOn|MidOff|      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_preonic_grid( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+  _______, RESET,   DEBUG,   _______, _______, _______, _______, TERM_ON, TERM_OFF,_______, _______, KC_DEL,  \
+  _______, _______, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, _______,  _______, _______, _______, _______, \
+  _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  _______, _______, _______, _______, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, TO(_QWERTY)  \
+)
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_startup[][2] = SONG(STARTUP_SOUND); 
+float tone_qwerty[][2] = SONG(QWERTY_SOUND);
+float tone_numpad[][2] = SONG(GOODBYE_SOUND);
+float tone_oneshot[][2] = SONG(PLANCK_SOUND);
+float tone_dyn_macro_rec[][2] = SONG(PREONIC_SOUND);
+float tone_dyn_macro_stop[][2] = SONG(COLEMAK_SOUND);
+float music_scale[][2] = SONG(DVORAK_SOUND);
+float tone_goodbye[][2] = SONG(TERMINAL_SOUND);
+float tone_goodbye1[][2] = SONG(WORKMAN_SOUND);
+float tone_goodbye2[][2] = SONG(PLOVER_SOUND);
+float tone_goodbye3[][2] = SONG(VOICE_CHANGE_SOUND);
+float tone_goodbye4[][2] = SONG(MINOR_SOUND);
+float tone_goodbye5[][2] = SONG(UNICODE_WINDOWS);
+float tone_goodbye6[][2] = SONG(PLOVER_GOODBYE_SOUND);
+float real_imp[][2]    = {
+  {NOTE_A4, 75}   ,{NOTE_REST, 25} ,{NOTE_A4, 75}   ,{NOTE_REST, 25} ,{NOTE_A4, 75}   ,{NOTE_REST, 25} ,
+  {NOTE_F4, 75}   ,{NOTE_C5, 25}   ,{NOTE_A4, 75}   ,{NOTE_REST, 25} ,
+  {NOTE_F4, 75}   ,{NOTE_C5, 25}   ,{NOTE_A4, 75}   ,{NOTE_REST, 25} ,
+  {NOTE_REST, 30}
+};
+
+#endif
+
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+        case QWERTY:
```

Comment:
Actually, it looks like this macro isn't do anything but playing a sound.  It could be removed.

Reply:
Thanks! I have made modifications based on your suggestions, this was really helpful :)